_posts/2022-12-21-easily-verify-your-rust-in-ci-with-kani.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: post
+title: "Easily verify your Rust in CI with Kani and Github Actions"
```

Comment:
Just want to say: I think this is a good title.

Reply:
👍